pkg/gui/view_helpers.go

```diff
@@ -139,6 +139,46 @@ func (gui *Gui) returnFocus(g *gocui.Gui, v *gocui.View) error {
 	return gui.switchFocus(g, v, previousView)
 }
 
+func (gui *Gui) goToStatus(g *gocui.Gui, v *gocui.View) error {
```

Comment:
You can DRY up this code by using a function that takes a view name and returns a function that will switch focus to that view name when called:
```
func (gui *Gui) createGoToSidePanelHandler(viewName string) func(g *gocui.Gui, v *gocui.View) error {
	return func(g *gocui.Gui, v *gocui.View) error {
		view, err := g.View(viewName)
		if err != nil {
			gui.Log.Error(err)
			return nil
		}
		return gui.switchFocus(g, nil, view)
	}
}

```

then in keybindings you can do:
```
Handler:  gui.createGoToSidePanelHandler("status"),
```

We could probably come up with a better name for the function but I'm happy with that :)

Reply:
Thanks for the tip, it allowed me to remove all the duplication :)

Reply:
I've also introduced a for loop to append the bindings, since we can get the Key casting the loop index into a rune